grails5 upgrade

niravassar opened the pull request.

Commit:
grails5 upgrade

sdelamo commented on complete/build.gradle:
```diff
@@ -5,8 +5,8 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
```
this should be decoupled from grails version

sdelamo commented on complete/build.gradle:
```diff
@@ -5,8 +5,8 @@ buildscript {
+        classpath "com.bertramlabs.plugins:asset-pipeline-gradle:3.3.4"
+        classpath "org.grails.plugins:hibernate5:7.1.0"
 
         classpath "org.grails.plugins:views-gradle:2.0.0.RC2"
```
this should be 2.1.1

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
```
grails version should be 5.0.1

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
+gormVersion=7.1.0.M5
```
this should be GORM 7.1.0

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
```
grails version should be 5.0.1

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
+gormVersion=7.1.0.M5
```
this should be GORM 7.1.0

jamesfredley committed:
Modernize CI to Java 21 + Gradle 9 baseline (refs apache/grails-static-website#354)

jamesfredley committed:
Revert "Modernize CI to Java 21 + Gradle 9 baseline (refs apache/grails-static-website#354)"

This reverts commit e8f7bac7e73a70872beb67d5632470e1969b37db.

jamesfredley committed:
Update README to point at canonical guide on grails.apache.org (refs apache/grails-static-website#354)

jamesfredley committed:
Replace dead publisher CI with sample-app gradle check workflow

Delete: githubactions-build.sh
Add: .github/workflows/gradle.yml

The previous CI invoked githubactions-build.sh which curled scripts from
grails/grails-guides to build now-stripped AsciiDoc and republish to the
deprecated guides.grails.org site. That entire pipeline is dead post-Phase
14b/15. The new workflow runs the sample app's own check task on a Java
version compatible with the branch (Java 11 for grails3/4/5, Java 17 for
grails6), using the repo's bundled gradlew without upgrading the wrapper.

(refs apache/grails-static-website#354)
